docs/content/topics/developers.md

```diff
@@ -108,297 +114,232 @@ seamlessly with Docker Desktop (Docker for Windows).
-```
-# Only if you are using Helm 2
-kubectl --namespace kube-system create serviceaccount tiller
-kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller 
-kubectl --namespace kube-system patch deploy tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}' 
+```console
+$ make hack-expose-apiserver
 ```
-- Run `DOCKER_ORG=brigadecore make build-all-images load-all-images` to build all images locally for the kind cluster
-- Run `make helm-install` to install/upgrade Brigade onto the kind cluster. This is the command you should re-run to test your changes during your Brigade development workflow. If this command does not work, you probably need to run `helm repo add brigade https://brigadecore.github.io/charts`
 
-When you're done, feel free to `kind delete cluster` to tear down the kind cluster resources.
+You can then log in to the apiserver with the following `brig` command:
 
-## Running Brigade inside remote Kubernetes
+```console
+$ brig login -s https://localhost:7000 -r -p 'F00Bar!!!' -k
+```
 
-Some developers use a remote Kubernetes instead of minikube.
+To create your first Brigade project, check out [projects](./projects.md) to
+see how it's done.
 
-To run a development version of Brigade inside of a remote Kubernetes,
-you will need to do two things:
+[registry addon docs]: https://minikube.sigs.k8s.io/docs/handbook/registry
 
-- Make sure you push your `brigade` docker images to a registry the cluster can access
-- Set the image when you do a `helm install brigade/<chart>` on the Brigade chart.
+## Kind configuration
 
-## Running Brigade (brigade-controller) Locally (against Minikube or kind)
+You can also use [kind] for your day-to-day Brigade development workflow.
+Kind has a great quickstart that can be found
+[here](https://kind.sigs.k8s.io/docs/user/quick-start/).
 
-Assuming you have Brigade installed (either on minikube or another cluster) and
-your `$KUBECONFIG` is pointing to that cluster, you can run `brigade-controller`
-locally.
+As the Brigade maintainers use kind heavily, there currently exists a helper
+script that will create a new kind cluster with a local private registry
+enabled, as well as setting up nfs as the local storage provisioner. To use
+this script, run:
 
 ```console
-$ ./bin/brigade-controller --kubeconfig $KUBECONFIG
+$ ./hack/kind/new-cluster.sh
 ```
 
-(The default location for `$KUBECONFIG` on UNIX-like systems is `$HOME/.kube`.)
+Now you're ready to build and push images to the local registry and deploy
+Brigade:
 
-For the remainder of this document, we will assume that your local `$KUBECONFIG`
-is pointing to the correct cluster.
+```console
+$ export DOCKER_REGISTRY=localhost:5000
+$ make hack
+```
 
-### Running the Functional Tests
+To expose the apiserver port, run the following command:
```

Comment:
If you use our hack script/target for creating a new kind cluster, we automatically map `localhost:31600` to the API server. Is it worth mentioning `make hack-expose-apiserver` as being a fallback if you're not running with our KinD configuration?